Echo/EchoHelp.cpp

```diff
@@ -63,7 +78,7 @@ void EchoHelp::printHelp(const std::vector< std::string > & helpFlags ,
 
     cout << "Usage: \n $polysync-echo [options] \n\n";
 
-    for( auto index = 0; index < 4; ++index )
+    for( auto index = 0; index < 5; ++index )
```

Comment:
Agreed, thank you for noticing that. Fixed / addressed here: https://github.com/PolySync/PolySync-CPP-Examples/commit/a28bcb83113b46af30b74c0c2a1267f802f798b9